src/main/app/idam/idamClient.ts

```diff
@@ -72,4 +72,9 @@ export class IdamClient {
         )
       })
   }
+
+  static invalidateCode (code: string): Promise<void> {
```

Comment:
Code is a bit misleading here especially that above method `exchangeCode` use the same vocabulary while the code in that case means totally different thing.

Please rename to `invalidateSession` or `invalidateUserAuthToken`.

Reply:
done